x-pack/plugins/security_solution/cypress/tasks/login.ts

```diff
@@ -291,7 +291,13 @@ export const getEnvAuth = (): User => {
 export const loginAndWaitForPage = (url: string, role?: ROLES) => {
   login(role);
   cy.visit(
-    `${url}?timerange=(global:(linkTo:!(timeline),timerange:(from:1547914976217,fromStr:'2019-01-19T16:22:56.217Z',kind:relative,to:1579537385745,toStr:now)),timeline:(linkTo:!(global),timerange:(from:1547914976217,fromStr:'2019-01-19T16:22:56.217Z',kind:relative,to:1579537385745,toStr:now)))`
+    `${url}?timerange=(global:(linkTo:!(timeline),timerange:(from:1547914976217,fromStr:'2019-01-19T16:22:56.217Z',kind:relative,to:1579537385745,toStr:now)),timeline:(linkTo:!(global),timerange:(from:1547914976217,fromStr:'2019-01-19T16:22:56.217Z',kind:relative,to:1579537385745,toStr:now)))`,
+    {
+      onBeforeLoad(win) {
+        // avoid cypress being held by windows prompt and timeout
```

Comment:
We test copy to clip board button on Network page. When Cypress runs locally for testing the functionality, the browser will be prompted (https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) as Cypress asks for accessing the clipboard.

Reply:
@angorayc I would avoid doing this globally since it may impact future and current tests. I think it is better to add a specific method for this and use it when needed :)

Reply:
ok, I'll pass a callback instead.